docs/mobile-apps/features/mobile-app-diagnostics/view-tree.md

```diff
@@ -0,0 +1,91 @@
+---
+id: view-tree
+title: Inspect View Tree
+sidebar_label: Inspect View Tree
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Inspect View Tree is a feature that is designed to help you discover all the visible elements of your mobile application while developing your test scripts. It captures a snapshot of the user interface and allows you to navigate and inspect view objects of your native apps.
```

Comment:
@kristofmuhi 
Small but maybe important note here. Can we add a note here that the view tree/ui hierarchy shown here is NOT the same as with Appium. They can't copy over the XPATH to find elements. Appium shows XML, this is not showing XML. Can we also add that the way Appium collects and translates the data is different in comparison to ours, but emphasise that our solution will help them debug if that element was actually in the UI Hierarchy

Reply:
Agreed! @aoleszyn  lets publish the doc in this form, overall looks good, and create a new PR for Wim's comments and figure out how to add this to the disclaimer part below